README.md

```diff
@@ -57,7 +58,7 @@ From maven central:
     ReliableKafkaConsumerPool<String,String> pool = new ReliablePoolBuilder<>(eventClientFactory)
                 .topics(List.of(TOPIC, TOPIC2)) // list of topics to subscribe to
                 .poolCount(5) // optional: number of threads consuming from kafka 
-                .afterProcess(consumer -> commit(consumer)) // optional: after a poll, do something with consumer
+                .afterProcess(consumer -> commit(consumer)) // optional: after a poll, do something with consumer. If you specify this, be sure to commit at some point in this lambda
```

Comment:
this will override the library's default commit, right? could add that to the comment?

Reply:
I was wondering if we should remove the option to do the after-processing lambda, since messing with this could possibly mess up committing the offset.

Reply:
Yeah, did a quick search in finn github - we don't use it. Might as well remove, and implement again if needed.

Reply:
ok, will do it in a different pr, created issue  #10